db/schema.rb

```diff
@@ -649,6 +687,21 @@
     t.index ["decidim_user_id"], name: "index_decidim_impersonation_logs_on_decidim_user_id"
   end
 
+  create_table "decidim_ldap_configurations", force: :cascade do |t|
```

Comment:
This should not be here, it belongs to DiBa only

Reply:
Ops, I don't know why is it here.